### PR TITLE
feat: add create refine app for cli

### PR DESCRIPTION
**Pull request by yildirayunlu**
Creating a new refine project with `refine-cli`.

Kapture 2022-11-22 at 16 28 40

### CODE REVIEW

File: packages/create-refine-app/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+import { Command } from "commander";
+import execa from "execa";
+
+const bootstrap = () => {
+    const program = new Command();
+    program
+        .version("0.0.0", "-v, --version", "Output the current version.")
```
Comment (aliemir) [inline packages/create-refine-app/src/index.ts:8]: Can we get the version from `package.json`?

Comment (yildirayunlu) [reply packages/create-refine-app/src/index.ts:8]: of course 🚀

File: packages/create-refine-app/.npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```
Comment (omeraplak) [inline packages/create-refine-app/.npmrc:1]: we shouldn't need this file

File: packages/create-refine-app/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+</div>
+
+## About
+
+[**refine**](https://refine.dev/) offers lots of out-of-the box functionality for rapid development, without compromising extreme customizability. Use-cases include, but are not limited to *admin panels*, *B2B applications* and *dashboards*.
+
+## Install
+
+```
+npm install @pankod/refine-sdk
```
Comment (omeraplak) [inline packages/create-refine-app/README.md:21]:
```suggestion
npm create refine
```

File: packages/create-refine-app/jest.config.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const { pathsToModuleNameMapper } = require("ts-jest/utils");
+const { compilerOptions } = require("./tsconfig");
+
+module.exports = {
+    preset: "ts-jest",
+    rootDir: "./",
+    name: "cli",
+    displayName: "cli",
+    setupFilesAfterEnv: ["<rootDir>/test/jest.setup.ts"],
+    testEnvironment: "jsdom",
```
Comment (omeraplak) [inline packages/create-refine-app/jest.config.js:10]:
```suggestion
    testEnvironment: "node",
```
Can we update according to node env?

File: packages/create-refine-app/package.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        "test": "jest --passWithNoTests --runInBand",
+        "prepare": "npm run build"
+    },
+    "bin": "./dist/index.js",
+    "devDependencies": {
+        "@esbuild-plugins/node-resolve": "^0.1.4",
+        "@testing-library/jest-dom": "^5.16.4",
+        "@testing-library/react": "^13.1.1",
+        "@testing-library/react-hooks": "^8.0.0",
+        "@testing-library/user-event": "^14.1.1",
```
Comment (omeraplak) [inline packages/create-refine-app/package.json:21]:
```suggestion
```
Can we update according to node env?

**Review by omeraplak — APPROVED** [review]

**Pull request merged**